.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: 💬 Community Discord Server (Login support requests go here!)
-    url: https://discord.gg/V6ckZEs
+    url: https://discord.gg/3Z9yzmA
```

Comment:
why did this need to be changed? it was already an non-expiring, unlimited uses invite to the discord server